pkg/config/property.go

```diff
@@ -49,4 +49,8 @@ const (
 
 	workspaceIdleTimeout        = "devworkspace.idle_timeout"
 	defaultWorkspaceIdleTimeout = "15m"
+
+	// Skip Verify For Insecure connection with TLS
```

Comment:
well, it actually disables verification for TLS (Secure) connections which could lead to usage of insecure connections.
And also it makes sense to add the same comment as go docs for that configuration has `This should be used only for testing.`
```suggestion
	// Skip Verify for TLS connections
	// It's insecure and should be used only for testing
```

Reply:
commented